From 9195c5d987da6074346d23fcf1ab90d29b5e3222 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:31 -0400
Subject: [redhat] kabi: Adding symbol genlmsg_put (net/netlink/genetlink.c)

Message-id: <61cd29a23043c01746bcb45d7b4b397133399938.1410510773.git.poros@redhat.com>
Patchwork-id: 95957
O-Subject: [PATCH RHEL7.1 091/233] kabi: Adding symbol genlmsg_put (net/netlink/genetlink.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/genlmsg_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/genlmsg_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/genlmsg_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/genlmsg_put b/redhat/kabi/kabi-module/kabi_ppc64/genlmsg_put
new file mode 100644
index 0000000..326ac7d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/genlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0xfe79d045 genlmsg_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/genlmsg_put b/redhat/kabi/kabi-module/kabi_s390x/genlmsg_put
new file mode 100644
index 0000000..06f73fc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/genlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0xa40df390 genlmsg_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/genlmsg_put b/redhat/kabi/kabi-module/kabi_x86_64/genlmsg_put
new file mode 100644
index 0000000..7ff64a6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/genlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0xdbce6f67 genlmsg_put vmlinux EXPORT_SYMBOL
-- 
1.7.1